### PR TITLE
monitor route creation time

### PR DESCRIPTION
**Pull request by zeitlinger**
fixes the init logic

### CODE REVIEW

**Comment by zeitlinger** [comment]
:+1:

File: filters/builtin/creationmetrics.go [hunk 1]
```diff
@@ -33,10 +33,17 @@ func NewRouteCreationMetrics(metrics filters.Metrics) *RouteCreationMetrics {
...
 	for _, r := range routes {
 		for origin, start := range m.startTimes(r) {
-			m.metrics.MeasureSince(metricsPrefix+origin, start)
+			if m.initialized {
+				m.metrics.MeasureSince(metricsPrefix+origin, start)
+			}
 		}
 	}
 
+	if !m.initialized {
```
Comment (tkrop) [inline filters/builtin/creationmetrics.go]: I would drop this condition and just set `initialized` always to `true`.

**Comment by tkrop** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**